news/_posts/core-weekly/2022-09-12-coreweekly-36-2022.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 36 of 2022"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2022-09-12
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 5th to Sunday 11th of September 2022.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+Last week we held the sixth Public Demo of the year. You can read more about it [in summary here](https://build.prestashop.com/news/public-demo-6-2022/). We encourage you to watch two demos and project news presented in that session.
```

Comment:
@matks, do you think about anything else important to inform?

Reply:
There is this information https://github.com/PrestaShop/PrestaShop/issues/26427#issuecomment-1239098983

Reply:
Maybe we could remind people that Hacktoberfest is coming? 😄

Reply:
I'll make sure to inform about Hacktoberfest a bit later when there will be a chance to sign up